cclib/parser/orcaparser.py

```diff
@@ -293,6 +323,47 @@ def splitter(line):
 
             self._append_scfvalues_scftargets(inputfile, line)
 
+        """
+-------------------------------------------------------------------------------
+                          DFT DISPERSION CORRECTION
+
+                              DFTD3 V3.1  Rev 1
+                              USING zero damping
+-------------------------------------------------------------------------------
+The omegaB97X-D3 functional is recognized. Fit by Chai et al.
+Active option DFTDOPT                   ...         3
+
+molecular C6(AA) [au] = 9563.878941
+
+
+            DFT-D V3
+ parameters
+ s6 scaling factor         :     1.0000
+ rs6 scaling factor        :     1.2810
+ s8 scaling factor         :     1.0000
+ rs8 scaling factor        :     1.0940
+ Damping factor alpha6     :    14.0000
+ Damping factor alpha8     :    16.0000
+ ad hoc parameters k1-k3   :    16.0000     1.3333    -4.0000
+
+ Edisp/kcal,au: -10.165629059768  -0.016199959356
+ E6   /kcal   :  -4.994512983
+ E8   /kcal   :  -5.171116077
+ % E8         :  50.868628459
+
+-------------------------   ----------------
+Dispersion correction           -0.016199959
+-------------------------   ----------------
+"""
+        if "DFT DISPERSION CORRECTION" in line:
+            # A bunch of parameters are printed the first time dispersion is called
+            # However, they vary wildly in form and number, making parsing problematic
+            line = next(inputfile)
+            while 'Dispersion correction' not in line:
+                line = next(inputfile)
```

Comment:
Should we do `self.skip_lines(inputfile,["Dispersion parameters"])``

Reply:
It isn't clear if the number of lines to skip is fixed, and we (I?) do this in other places.